maven-plugin/src/main/java/com/spotify/missinglink/maven/CheckMojo.java

```diff
@@ -550,56 +427,16 @@ private static long asMillis(Stopwatch stopwatch) {
     return stopwatch.elapsed(TimeUnit.MILLISECONDS);
   }
 
-  private void outputConflicts(Collection<Conflict> conflicts) {
+  private void outputConflicts(Collection<Conflict> conflicts, CheckReport checkReport) {
```

Comment:
feels awkward to me to pass in the CheckReport as a parameter if this method is only called from one place, seems like the signature should just be `CheckReport outputConflicts(conflicts)`